src/JSInterop/Microsoft.JSInterop/src/IJSVoidResult.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.JSInterop
+{
+    /// <summary>
+    /// Represents a void result from a JavaScript call.
```

Comment:
Could we add the boiler plate that says this is infrastructure and should not be used by user code